README.md

```diff
@@ -1,11 +1,11 @@
 # Closure Rules for Bazel (αlpha) [![Build Status](https://travis-ci.org/bazelbuild/rules_closure.svg?branch=master)](https://travis-ci.org/bazelbuild/rules_closure)
 
-JavaScript | Templating | Stylesheets
---- | --- | ---
-[closure_js_library](#closure_js_library) | [closure_template_js_library](#closure_template_js_library) | [closure_css_library](#closure_css_library)
-[closure_js_binary](#closure_js_binary) | [closure_template_java_library](#closure_template_java_library) | [closure_css_binary](#closure_css_binary)
-[closure_js_deps](#closure_js_deps) | [closure_template_py_library](#closure_template_py_library) |
-[closure_js_test](#closure_js_test) | |
+JavaScript | Templating | Stylesheets | Protocol Buffers
+--- | --- | --- | ---
+[closure_js_library](#closure_js_library) | [closure_template_js_library](#closure_template_js_library) | [closure_css_library](#closure_css_library) | [closure_proto_js_library](#closure_proto_js_library)
```

Comment:
FYI: I've been meaning to test a theory I have that could potentially allow me to integrate `closure_template_{js,java,py}_library` into a single `closure_template_library` statement. If I'm able to pull that off, the design of this will be changed.